Looks like the Windows issue is still there. And GCC/Clang has the same issue
```
2026-02-23T20:36:13.6011166Z /home/runner/work/amrex/amrex/Tests/ReduceToPlanePatchy/main.cpp:70:25: error: capturing a structured binding is not yet supported in OpenMP [clang-diagnostic-error]
2026-02-23T20:36:13.6012576Z    70 |         for (MFIter mfi(plane_unique); mfi.isValid(); ++mfi) {
2026-02-23T20:36:13.6013127Z       |                         ^
2026-02-23T20:36:13.6014464Z /home/runner/work/amrex/amrex/Tests/ReduceToPlanePatchy/main.cpp:54:28: note: 'plane_unique' declared here
2026-02-23T20:36:13.6015703Z    54 |         auto [plane_patch, plane_unique] = ReduceToPlaneMF2Patchy<ReduceOpSum>(dir, domain, mf,
2026-02-23T20:36:13.6016495Z       |                            ^
2026-02-23T20:36:13.6017869Z /home/runner/work/amrex/amrex/Tests/ReduceToPlanePatchy/main.cpp:71:30: error: capturing a structured binding is not yet supported in OpenMP [clang-diagnostic-error]
2026-02-23T20:36:13.6019299Z    71 |             auto const& pa = plane_unique.const_array(mfi);
2026-02-23T20:36:13.6030097Z       |                              ^
2026-02-23T20:36:13.6031188Z /home/runner/work/amrex/amrex/Tests/ReduceToPlanePatchy/main.cpp:54:28: note: 'plane_unique' declared here
2026-02-23T20:36:13.6032510Z    54 |         auto [plane_patch, plane_unique] = ReduceToPlaneMF2Patchy<ReduceOpSum>(dir, domain, mf,
2026-02-23T20:36:13.6033281Z       |                            ^
```

--- a/Src/Base/AMReX_MultiFabUtil.H
+++ b/Src/Base/AMReX_MultiFabUtil.H
@@ -471,7 +471,8 @@ namespace amrex
      * \tparam F   callable type like a lambda function
      *
      * \param direction normal direction of the plane (e.g., 0, 1 and 2)
-     * \param domain    domain Box
+     * \param domain    domain Box. For patchy inputs this must cover the full
+     *                  extent of mf.boxArray() (subdomains are not supported).
      * \param mf        a FabArray/MultiFab object specifying the iteration space
      * \param f         a callable object returning data for reduction. It takes
      *                  four ints, where the first int is the local box index and
@@ -1457,12 +1458,15 @@ template <typename Op, typename FA, typename F,
 std::pair<FA,FA>
 ReduceToPlaneMF2Patchy (int direction, Box const& domain, FA const& mf, F const& f)
 {
-    using T = typename FA::value_type;
-
-    T initval;
-    Op().init(initval);
+    AMREX_ALWAYS_ASSERT_WITH_MESSAGE(mf.is_cell_centered(), "ReduceToPlaneMF2Patchy: only cell-centered data is supported.");
 
     Box const ndomain = amrex::convert(domain, mf.ixType());
+    AMREX_ALWAYS_ASSERT_WITH_MESSAGE(ndomain.contains(mf.boxArray().minimalBox()),
+        "ReduceToPlaneMF2Patchy: subdomains are not supported; domain must cover the full extent of mf.boxArray().");
+
+    using T = typename FA::value_type;
+    T initval;
+    Op().init(initval);
 
     BoxList bl_patch2d = mf.boxArray().boxList();
     for (auto& b : bl_patch2d) {

--- a/Tests/ReduceToPlanePatchy/GNUmakefile
+++ b/Tests/ReduceToPlanePatchy/GNUmakefile
@@ -1,0 +1,24 @@
+AMREX_HOME := ../..
+
+DEBUG	= FALSE
+
+DIM	= 3
+
+COMP    = gcc
+
+USE_MPI   = FALSE
+USE_OMP   = FALSE
+USE_CUDA  = FALSE
+USE_HIP   = FALSE
+USE_SYCL  = FALSE
+
+BL_NO_FORT = TRUE
+
+TINY_PROFILE = FALSE
+
+include $(AMREX_HOME)/Tools/GNUMake/Make.defs
+
+include ./Make.package
+include $(AMREX_HOME)/Src/Base/Make.package
+
+include $(AMREX_HOME)/Tools/GNUMake/Make.rules

--- a/Tests/ReduceToPlanePatchy/Make.package
+++ b/Tests/ReduceToPlanePatchy/Make.package
@@ -1,0 +1,1 @@
+CEXE_sources += main.cpp

--- a/Tests/ReduceToPlanePatchy/main.cpp
+++ b/Tests/ReduceToPlanePatchy/main.cpp
@@ -28,9 +28,6 @@ int main (int argc, char* argv[])
         DistributionMapping dm(ba);
         MultiFab mf(ba, dm, 1, 0);
 
-#ifdef AMREX_USE_OMP
-#pragma omp parallel if (Gpu::notInLaunchRegion())
-#endif
         for (MFIter mfi(mf); mfi.isValid(); ++mfi) {
             auto const& a = mf.array(mfi);
             Box const& bx = mfi.validbox();
@@ -64,9 +61,6 @@ int main (int argc, char* argv[])
 
         // Compare unique sparse result to reference plane on overlapping cells.
         Real max_err = 0.0;
-#ifdef AMREX_USE_OMP
-#pragma omp parallel if (Gpu::notInLaunchRegion()) reduction(max:max_err)
-#endif
         for (MFIter mfi(plane_unique); mfi.isValid(); ++mfi) {
             auto const& pa = plane_unique.const_array(mfi);
             Box const& bx = mfi.validbox();

